.gitignore

```diff
@@ -15,7 +15,7 @@
 # vendor/
 
 # Ignore binary
-./kam
+dist/
 kam_windows_amd64.exe
 kam_linux_amd64
 kam_darwin_amd64
```

Comment:
Adding of individual binary is not required if you are adding the parent directory, so remove the distribution binaries